common/static/common/templates/discussion/forum-action-vote.underscore

```diff
@@ -2,7 +2,7 @@
     <span aria-hidden="true" class="display-vote is-hidden" >
       <span class="vote-count"></span>
     </span>
-    <a href="#" class="action-button action-vote" role="checkbox" aria-checked="false">
+    <button class="btn-link action-button action-vote" role="checkbox" aria-checked="false">
```

Comment:
@clrux and I discussed this and we're going to leave it as is for now. I've made a JIRA ticket to add toggle buttons to the pattern library:

https://openedx.atlassian.net/browse/FEDX-238